### ISSUE TITLE
Broken link: image_conditioning / Style Model

### ISSUE THREAD

**Issue by sahuhtala** [cubiq/ComfyUI_Workflows#1]
The link "CLIP Vision model" seems broken in the text on page Image Conditioning / Style model.  It was pretty hard to follow what to actually do.

I suppose the correct URL to link is this: https://huggingface.co/openai/clip-vit-large-patch14/tree/main/
**pytorch_model.bin**

It might also be a good idea to link to the _style model_ that is mentioned in the text:
https://huggingface.co/TencentARC/T2I-Adapter/tree/main/models/
For example **coadapter-style-sd15v1.pth** in case of SD1.5 

Otherwise very informative article, thank you!

**Comment by cubiq** [comment]
yeah forgot about those, thanks for the headsup!

fixed!
